### PR TITLE
XRT-3111: Update opc ua server federated example

### PR DESCRIPTION
**Pull request by BinduRao2018**

### CODE REVIEW

File: Servers/opc-ua/federated/deployment/config_securitypolicy/opc-ua-server.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "Library": "libxrt-opc-ua-server.so",
+    "Factory": "xrt_opc_ua_server_factory",
+    "ThreadPool": "pool",
+    "Bus": "bus",
+    "Logger": "logger",
+    "ApplicationUri" : "urn:open62541.server.application",
```
Comment (dpazj) [inline Servers/opc-ua/federated/deployment/config_securitypolicy/opc-ua-server.json]: Do we want to use this application uri in the example or should we have it set to something related to our product? If so then we will need to update the parameters to the create_certificate script

**Review by dpazj — COMMENTED** [review]
One comment to discuss

**Comment by BinduRao2018** [comment]
@dpazj, addressed review comment

**Review by dpazj — APPROVED** [review]
LGTM

**Pull request merged**